practice-one/src/index.html

```diff
@@ -38,13 +35,48 @@
         <a class="btn btn-link btn-help-center" href="#">
           Help Center
         </a>
-
       </nav>
-
     </div>
-
   </header>
+  <!-- END HEADER -->
+
+  <!-- START MAIN -->
+  <main class="main-content">
+    <div class="container">
+      <!-- START SECTION GET-STARTED -->
+      <section class="section-get-started">
+        <section class="download">
+          <h1 class="download-heading">
+            Download app,
+            save money, make
+            friends!
+          </h1>
+          <p class="content download-content">
+            Its simple and its free. Play your part in reducing Carbon Footprint and help
+            Mother Nature to sustain its beauty. So what are you waiting for? Lets ride
+            together
+          </p>
 
+          <div class="download-links">
+            <a href="#" class="btn btn-link btn-download">
+              <img src="./assets/images/download-icon-store.svg" alt="download-icon-store">
+              Download
+            </a>
+            <a href="#" class="btn btn-link btn-download">
+              <img src="./assets/images/download-icon-figma.svg" alt="download-icon-figma">
+              Download
+            </a>
+          </div>
+        </section>
+
+        <figure class="background-img">
+          <img src="./assets/images/download-background.png" alt="background-img">
```

Comment:
Please help to update the class name, file name, and alt value for meaning. You can use download-thumbnail, download-image....

Reply:
I will remove class name of figure for not using it and update the others.